docs/userguide/configuring.rst

```diff
@@ -48,13 +48,46 @@ used by the infiniband interface with ``address_by_interface('ib0')``
 
 .. contents:: Configuration How-To and Examples:
 
-.. note::
-   All configuration examples below must be customized for the user's 
-   allocation, Python environment, file system, etc.
+
+Creating and Using Config Objects
+---------------------------------
+
+:class:`~parsl.config.Config` objects are loaded to define the "Data Flow Kernel" (DFK) that will manage tasks.
+All Parsl applications start by creating or importing a configuration then calling the load function.
+
+.. code-block:: python
+
+    from parsl.configs.htex_local import config
+    import parsl
+
+    parsl.load(config)
+
+The ``load`` statement can happen after Apps are defined and must occur before tasks are started.
+
```

Comment:
Grammar wise, suggest changing the conjunction to `but`:
```diff
- The ``load`` statement can happen after Apps are defined and must occur before tasks are started.
+ The ``load`` statement can happen after Apps are defined but must occur before tasks are started.
```

Reply:
Done :)